### ISSUE TITLE
False positives from loadInclude rule when $name argument includes unparseable variable

### ISSUE THREAD

**Issue by Boegie** [mglaman/phpstan-drupal#547]
# Bug report
Whenever the $name argument of `loadInclude($module, $type, $name = NULL)` includes a variable we can't parse, currently we return a false positive.

### Code snippet that reproduces the problem
For example, this code in `core/modules/system/tests/modules/entity_test/entity_test.install`
```
$module_handler = \Drupal::moduleHandler();
$index = \Drupal::state()->get('entity_test.db_updates.entity_definition_updates');
$module_handler->loadInclude('entity_test', 'inc', 'update/entity_definition_updates_' . $index);
$index = \Drupal::state()->get('entity_test.db_updates.status_report');
$module_handler->loadInclude('entity_test', 'inc', 'update/status_report_' . $index);
```
returns
```
		-
			message: "#^File core/modules/system/tests/modules/entity_test/entity_test\\.inc could not be loaded from Drupal\\\\Core\\\\Extension\\\\ModuleHandlerInterface\\:\\:loadInclude$#"
			count: 2
			path: modules/system/tests/modules/entity_test/entity_test.install
```
